tests/foreman/sys/test_katello_certs_check.py

```diff
@@ -21,6 +21,54 @@
 
 import pytest
 
+import robottelo.constants as constants
+from robottelo.config import settings
+
+
+@pytest.mark.e2e
+def test_positive_install_sat_with_katello_certs(certs_data, sat_ready_rhel):
+    """Install Satellite with custom certs.
+
+    :id: 47e3a57f-d7a2-40d2-bbc7-d1bb3d79a7e1
+
+    :steps:
+
+        1. Generate the custom certs on RHEL machine
+        2. Install satellite with custom certs
+        3. Assert output does not report SSL certificate error
+        4. Assert all services are running
+
+
+    :expectedresults: Satellite should be installed using the custom certs.
+
+    :CaseAutomation: Automated
+    """
+    version = sat_ready_rhel.os_version.major
+    sat_ready_rhel.download_repofile(product='satellite', release=settings.server.version.release)
+    sat_ready_rhel.register_to_cdn()
+    for repo in getattr(constants, f"OHSNAP_RHEL{version}_REPOS"):
+        sat_ready_rhel.enable_repo(repo, force=True)
+    sat_ready_rhel.execute('yum -y update')
+    result = sat_ready_rhel.execute(
+        'dnf -y module enable satellite:el8 && dnf -y install satellite'
+    )
+    assert result.status == 0
+    command = (
+        'satellite-installer --scenario satellite '
+        f'--certs-server-cert "/root/{certs_data["cert_file_name"]}" '
+        f'--certs-server-key "/root/{certs_data["key_file_name"]}" '
+        f'--certs-server-ca-cert "/root/{certs_data["ca_bundle_file_name"]}" '
+    )
+    result = sat_ready_rhel.execute(command, timeout=2200000)
+    assert result.status == 0
+    # assert no hammer ping SSL cert error
+    result = sat_ready_rhel.execute('hammer ping')
+    assert 'SSL certificate verification failed' not in result.stdout
+    assert result.stdout.count('ok') == 8
```

Comment:
I think it is not a good idea to hard code count of ok's, better to use approch to count this similar to cli/test_ping.py

Reply:
Enhanced check to match `Status:` and `ok` counts